app/controllers/redirections_controller.rb

```diff
@@ -0,0 +1,13 @@
+class RedirectionsController < ApplicationController
+  def next
+    redirection = Redirection.find_by(slug: params[:slug])
```

Comment:
Hmmm interesting, I had the thought that we would be registering them and setting the order. This is a whole new twist...